tests/integration-tests-topologies/src/main/java/org/apache/pulsar/tests/topologies/PulsarCluster.java

```diff
@@ -260,6 +312,12 @@ public ExecResult runAdminCommandOnAnyBroker(String...commands) throws Exception
         return container.execCmd(cmds);
     }
 
+    public ExecResult createTenanantName(String tenantName, String clusterName, String roleName) throws Exception {
```

Comment:
this method is not used by any tests. better to introduce it when using it.

Reply:
will move it the test